<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>882</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>882</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MapleSeed++</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>595</width>
      <height>415</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QListWidget" name="libraryListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>371</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Database</string>
     </attribute>
     <widget class="QLineEdit" name="searchInput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>501</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="databaseListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>571</width>
        <height>341</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="regionBox">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="maxCount">
       <number>10</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>USA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EUR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JAP</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Download Queue</string>
     </attribute>
     <widget class="QTableWidget" name="downloadQueueTableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>371</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Help</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>281</width>
        <height>141</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Up -&gt; DPad-Up</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Down -&gt; DPad-Down</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prev Tab -&gt; L1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Next Tab -&gt; R1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Start Game -&gt; Button A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Close Game -&gt; L1+L2+R1+R2+Select</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>861</width>
      <height>28</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
border: 1px solid black;
text-align: center;
padding: 1px;
width: 15px;
}

QProgressBar::chunk {
background-color: #cd9bff;
border: 1px solid black;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="coverLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>25</y>
      <width>271</width>
      <height>386</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLog">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Log</string>
     </property>
     <addaction name="actionDebug"/>
     <addaction name="actionOpenLog"/>
    </widget>
    <addaction name="menuLog"/>
    <addaction name="actionClearSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCemu">
    <property name="title">
     <string>Cemu</string>
    </property>
    <widget class="QMenu" name="menuLibrary">
     <property name="title">
      <string>Library</string>
     </property>
     <addaction name="actionCemuChangeLibrary"/>
     <addaction name="actionCemuRefreshLibrary"/>
    </widget>
    <widget class="QMenu" name="menuContent_2">
     <property name="title">
      <string>Content</string>
     </property>
     <addaction name="actionCemuDownload"/>
     <addaction name="actionCemuDecrypt"/>
    </widget>
    <addaction name="actionCemuIntegrate"/>
    <addaction name="actionCemuFullscreen"/>
    <addaction name="separator"/>
    <addaction name="menuLibrary"/>
    <addaction name="menuContent_2"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionGamepad"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCemu"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCemuChangeLibrary">
   <property name="text">
    <string>Change</string>
   </property>
   <property name="toolTip">
    <string>Change the current games location</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionCemuDecrypt">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </action>
  <action name="actionCemuIntegrate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Integrate</string>
   </property>
  </action>
  <action name="actionCemuRefreshLibrary">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionClearSettings">
   <property name="text">
    <string>Clear Settings</string>
   </property>
  </action>
  <action name="actionCemuDownload">
   <property name="text">
    <string>Download</string>
   </property>
   <property name="toolTip">
    <string>Download content using a supplied id</string>
   </property>
  </action>
  <action name="actionGamepad">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gamepad</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionOpenLog">
   <property name="text">
    <string>Open Log</string>
   </property>
  </action>
  <action name="actionCemuFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
